.buildkite/pipelines/on_merge.yml

```diff
@@ -109,20 +109,6 @@ steps:
         - exit_status: '-1'
           limit: 3
 
-  - command: .buildkite/scripts/steps/checks/capture_oas_snapshot.sh
-    label: 'Check OAS Snapshot'
-    agents:
-      image: family/kibana-ubuntu-2004
-      imageProject: elastic-images-prod
-      provider: gcp
-      machineType: n2-standard-4
-      preemptible: true
-    timeout_in_minutes: 60
-    retry:
-      automatic:
-        - exit_status: '-1'
-          limit: 3
-
```

Comment:
I don't think it makes sense to run this as part of on_merge pipeline... We are basically spinning up ES and Kibana 2x to try and capture/commit the new snapshots except this script is specifically written to avoid committing the new snapshot when not running on a PR: https://github.com/elastic/kibana/blob/6c2cbf10a6f6d11c80e7333af5e106a5cc007f89/.buildkite/scripts/steps/checks/capture_oas_snapshot.sh#L25

Reply:
> I don't think it makes sense to run this as part of on_merge pipeline...

The 8 minutes isn't a huge cost if there's utility.  I'm less familiar with the surface area of errors, if you think it's okay this works for me.  There were occasional errors a few months back, e.g.  https://buildkite.com/elastic/kibana-on-merge/builds/54955#01932af7-13fa-4e7f-a4a5-32a4be549abd

> Or do we wanna turn on the file changing?

I'm hesitant to turn on auto-commits on main without CI and reviews in between.  We could probably find a way to open fix pull requests if needed.